project.clj

```diff
@@ -126,7 +126,7 @@
              :ezbake {:dependencies ^:replace [[puppetlabs/puppetdb ~pdb-version]
                                                [org.clojure/tools.nrepl "0.2.3"]]
                       :name "puppetdb"
-                      :plugins [[puppetlabs/lein-ezbake "0.2.9"
+                      :plugins [[puppetlabs/lein-ezbake "0.3.7"
```

Comment:
With this change it should be safe to pull lines 141 and 142 (assuming we want PE and FOSS on the same ezbake version.